src/components/atoms/SocialButton/SocialButton.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactComponent as GithubLogo } from 'assets/github-logo.svg';
```

Comment:
<img width="621" alt="Screenshot 2020-08-31 at 12 11 59" src="https://user-images.githubusercontent.com/19636010/91709649-5525b880-eb83-11ea-9da1-aefc5da25b7a.png">
<img width="546" alt="Screenshot 2020-08-31 at 12 12 05" src="https://user-images.githubusercontent.com/19636010/91709653-56ef7c00-eb83-11ea-98a6-0ef632ad08f6.png">

Reply:
Both are not working correctly. We cannot see logo. Try to add logos and text by default and remove controls